linux/inc/c_pal/platform_linux.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef PLATFORM_LINUX_H
+#define PLATFORM_LINUX_H
+
+#include "umock_c/umock_c_prod.h"
+
+#include "c_pal/completion_port_linux.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+    MOCKABLE_FUNCTION(, COMPLETION_PORT_HANDLE, platform_get_completion_port);
```

Comment:
>platform_get_completion_port [](http://example.com/codeflow?start=48&length=28)

what happened to platform_init/deinit which are listed in the .md?

Reply:
Those are in the platform.h in the interface folder.  This is for linux only